var/ramble/repos/builtin/base_applications/openfoam/base_application.py

```diff
@@ -421,6 +421,13 @@ class Openfoam(ExecutableApplication):
         file="{experiment_run_dir}/log.simpleFoam",
     )
 
+    success_criteria(
+        "nan",
+        mode="string",
+        anti_match=r".*[+-=\s]nan",
```

Comment:
```suggestion
        anti_match=r".*?[+-=\s]nan",
```
Good idea! FWIW if you do it like this with a "suggestion" block I think there is a cool "one click accept"